docs/src/tutorials/tutorial_5.md

```diff
@@ -0,0 +1,377 @@
+
+```julia
+...
+        Action(
+	    :atom,
+            (args...) -> element(),
+            subactions = [
+                SubAction(1:300, Scaling(1, 30)),
+                SubAction(301:350, Scaling(30, 16)),
+            ],
+        ),
+	...
+```
+
+Now, we must update the `element` function to return the current scale of the circle being drawn.
+The reason why we want the scale of the object is that the scale of the circle represents its atomic mass.
+At this time, `Javis.jl` does not allow a user to get the current scale of an object directly.
+This will most likely be changed in future versions.
+The following changes need to be made to the `element` function:
+
+```julia
+function element(p = O, color = "black")
+    sethue(color)
+    c = circle(p, 1, :fill)
+    return val(:_current_scale)
+end
+```
+
+Now, whenever the `Action` executes in `javis`, to draw the circle, the current scale of the circle object is stored into the `:atom` symbol.
+Great! But... How do we know what element it is we are looking at?
+
+To answer that question, we need to create another function to display this information!
+
+## What's That Element? 🔍
+
+To identify the element and display its information properly, let's create an info box similar to what we made in [Tutorial 2](tutorial_2.md#As-You-Can-See-Here...)!
+We do this by creating an `info_box` function that takes in a value:
+
+```julia
+function info_box(args...; value = 0)
+    fontsize(12)
+    box(140, -210, 170, 40, :stroke)
+    for element in elements
+        if value == round(ustrip(element.atomic_mass))
+            text(
+                "Element name: $(element.name)",
+                140,
+                -220,
+                valign = :middle,
+                halign = :center,
+            )
+            text(
+                "Atomic Mass: $(round(ustrip(element.atomic_mass)))",
+                140,
+                -200,
+                valign = :middle,
+                halign = :center,
+            )
+        end
+    end
+end
+```
+
+Whenever a mass is measured that corresponds to a known element, the name of that element will now be displayed.
+Of course, we need to further update our `javis` function to this:
+
+```julia
+...
+        Action(
+            :atom,
+            (args...) -> element(),
+            subactions = [
+                SubAction(1:300, Scaling(1, 30)),
+                SubAction(301:350, Scaling(30, 16)),
+            ],
+        ),
+        Action(1:300, (args...) -> info_box(value = round(val(:atom)[1]))),
+        Action(351:500, (args...) -> info_box(value = round(val(:atom)[1]))),
+    ],
+    pathname = "atomic.gif",
+    framerate = 10,
+...
+```
+
+The current scale of the circle object is now passed to the `info_box` function via the `:atom` symbol.
+This produces the following animation:
+
+![](assets/min_atomic_info.gif)
```

Comment:
Can you have a look at this again? The info box is flickering. Should that be the case? It looks a bit weird to me.

Reply:
Fixed the flickering